src/lib/containers/query_wrapper_plot_nav/QueryWrapperPlotNav.stories.tsx

```diff
@@ -203,3 +207,16 @@ DatasetCollection.args = {
   hideSqlEditorControl: false,
   shouldDeepLink: false,
 }
+
+export const People = Template.bind({})
```

Comment:
Added a People rendering to demo this exploration from the AD portal for https://sagebionetworks.jira.com/browse/PORTALS-2431